s4e-backend/src/main/java/pl/cyfronet/s4e/controller/SearchController.java

```diff
@@ -0,0 +1,44 @@
+package pl.cyfronet.s4e.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pl.cyfronet.s4e.controller.request.ZoneParameter;
+import pl.cyfronet.s4e.controller.response.SearchResponse;
+import pl.cyfronet.s4e.service.SearchService;
+import pl.cyfronet.s4e.util.TimeHelper;
+
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static pl.cyfronet.s4e.Constants.API_PREFIX_V1;
+
+@RestController
+@RequestMapping(path = API_PREFIX_V1, produces = APPLICATION_JSON_VALUE)
+@RequiredArgsConstructor
+@Tag(name = "search", description = "The Scene API")
+public class SearchController {
+    private final SearchService searchService;
+    private final TimeHelper timeHelper;
+
+    @Operation(summary = "View a list of scenes")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Successfully retrieved list")
+    })
+    @GetMapping("/search")
+    public List<SearchResponse> getScenes(@RequestParam Map<String, Object> params) throws SQLException {
+        ZoneParameter tz = (ZoneParameter) params.getOrDefault("tz", "UTC");
```

Comment:
A better option for `tz` variable would be `timeZone` or `utcTimeZone`

Reply:
i would go with timeZone.
i followed convention from SceneController.getScenes. 
@jswk was there a reason for 'tz'?

Reply:
Yup, we have `tz` parameter in SceneController already, so @kmarszalek is correct here following it. Btw, `utcTimeZone` doesn't make sense at all.

Reply:
what about timeZone?

Reply:
Could be `timeZone`, but then it should be changed in other places too.